server/lib/service/google_api.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:async';
+
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:http/http.dart' as http;
+
+/// A class that provides authenticated clients to interact with Google APIs.
+class GoogleApi {
+  /// The scopes the service account client needs to access Cloud Scheduler
+  /// and Firestore.
+  static const List<String> serviceAccountScope = [
+    'https://www.googleapis.com/auth/cloud-platform',
+    'https://www.googleapis.com/auth/cloud-scheduler',
+    'https://www.googleapis.com/auth/datastore',
+  ];
+
+  /// The scopes the user account client needs to access the ID token and
+  /// DV360 API.
+  static const List<String> userAccountScope = [
+    'openid email',
```

Comment:
is this needed?

Reply:
You're right, oops, my mistake. I forgot to remove it out when I separated the server from the client.